services/src/performance-tests/src/test/groovy/org/openkilda/performancetests/spec/EnduranceSpec.groovy

```diff
@@ -99,9 +99,69 @@ idle. Step repeats pre-defined number of times"
         topoHelper.purgeTopology(topo)
     }
 
-    def createFlow() {
+    def "Create 4094 flows"() {
+        // system allows to create 4094 simple flows or 2047 protected flows
+        def switchesAmount = 7
+        int islsAmount = switchesAmount * 2.5
+        int flowsAmount = 4094
+
+        setup: "Create a topology"
+        def topo = topoHelper.createRandomTopology(switchesAmount, islsAmount)
```

Comment:
I have couple thoughts here. This topology creation algo does not guarantee that all switches are interconnected. This may lead to some of your flow creation attempts to throw 'no path found' error.
I think we should look towards implementing typical topology type algos